src/main/java/wolox/training/models/User.java

```diff
@@ -17,7 +17,7 @@
     @Column(nullable = false)
     private LocalDate birthdate;
 
-    @OneToMany
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
This is incorrect. Because by using this CascadeType, it will try to persist and delete any books that are added to the collection, but from the database itself.
The behaviour should be that it only updates the relationship (and therefore, only the join table that get's generated with the ManyToMany.